<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QWidget" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Credit</string>
  </property>
  <widget class="QFrame" name="input_layout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 209, 246); color: black;
    </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>49</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Loan amount:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amount_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="period_val">
       <property name="styleSheet">
        <string notr="true">background-color: white;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>131</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Interest rate per year</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="interest_rate">
       <property name="styleSheet">
        <string notr="true">background-color: white; text-align: right;
        </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>186</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="annuity">
       <property name="text">
        <string>annuity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="diff">
       <property name="text">
        <string>differentiated</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 18px;</string>
    </property>
    <property name="text">
     <string>Loan calculator</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { color: #000; border-bottom: 1px solid black; border-radius:  5px; background-color: rgb(150, 167, 255); }
        QPushButton::pressed {
        background-color: grey;}
     </string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>241</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(150, 172, 255); color: black;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="overpayment">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Overpayment on credit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="overpayment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { qproperty-alignment: AlignCenter;}
        </string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="total_payments">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Total payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="total_payment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { qproperty-alignment: AlignCenter;}
        </string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="monthly_payments">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">qproperty-alignment: AlignCenter;</string>
       </property>
       <property name="text">
        <string>Monthly payments</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="monthly_payment_val">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { qproperty-alignment: AlignCenter; }
        </string>
       </property>
       <property name="text">
        <string>00.00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
